### ISSUE TITLE
How to apply LRP from Innvestigate package to 3D CNN regression problem?

### ISSUE THREAD

**Issue by diptiSH** [albermax/innvestigate#279]
I am using 3D CNN regression for the analysis of fog occurrences over the Indo-gangetic plains of India using CMIP6 model outputs. I want to check the feature importances of different variables over different regions for particular occurrences of fog. Can we apply LRP from innvestigate package to 3D CNN?   I could find all the examples of classification, therefore I am asking this question.

**Comment by adrhill** [comment]
Hi @diptiSH,

I'm not familiar with the CMIP6 model, but models using Kera's `Conv3D` layers should be supported by iNNvestigate. However—as you have noticed—iNNvestigate's focus is on classifiers. 
I'm pinging @sltzgs who is actively researching LRP for regression problems, maybe he can help you out.

**Comment by sltzgs** [comment]
Hi @diptiSH,

Indeed, most of the XAI work has evolved around explaining classifiers. That is why we have started to focus on explaining regression models in our latest research. As a starting point, you might find [our paper on the topic](https://arxiv.org/abs/2112.11407) useful. Moreover, we have created a [homepage dedicated to XAI for regression](xai-regression.org) where we summarize our findings and collect useful literature. Some code will also be published alongside in the coming months. In case you have further questions or want to bring some literature to our attention, feel free to reach out.

Cheers,
Simon
